### PR TITLE
Gitignore update

### PR DESCRIPTION
**Pull request by bejalane**
Added some new files to gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+somefile.js
```
Comment (RudeMax) [inline .gitignore:26]: Looks suspicious. No future in codding

File: .gitignore [hunk 2]
```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
```
Comment (MuscularSloth) [inline .gitignore:24]: **Hello!** Test comment

**Review by MuscularSloth — CHANGES_REQUESTED** [review]
Request changes

**Review by RudeMax — CHANGES_REQUESTED** [review]
Terrible.

**Review by RudeAnna — APPROVED** [review]
This looks great

**Review by KenAdams-AK — APPROVED** [review]
Approved

**Review by RudeMax — APPROVED** [review]
Nice! That's what Im talking about!

**Review by MuscularSloth — APPROVED** [review]
Great!

**Pull request merged**